### ISSUE TITLE
Autosave

### ISSUE THREAD

**Issue by stevekrouse** [stevekrouse/WoofJS#487]
Take off where #352 left off. 

Every X changes, save the currentVersion.

Every Y changes or Z minutes, save the currentVersion as a version.

**Comment by truthgoddess** [comment]
@stevekrouse I think we should prioritize this. I find, as I'm making WoofJS games, a problematic pattern. If I don't save the project I am working on, after long enough, woof will crash, and auto refresh to the last saved version. That forces me to click the save button almost after every change. I'd rather it just autosaved as you described above.

**Comment by stevekrouse** [comment]
Agreed this is a priority. Do you have time to take it on or just want me to bump it up on my priorities? @madduccino I wonder how it'd fit in the list of priorities we made here https://docs.google.com/document/d/1wF0YTupCKD61SGGOV5cp1luyzllbIZFo0icWFyDUKis/edit#

**Comment by truthgoddess** [comment]
I will have some time to take it on the week of August 19th (if @madduccino wants me to), or after September 1st.
